src/parser/reference-parser.ts

```diff
@@ -147,6 +162,18 @@ export function parseColumnName(column: AnyColumn<any, any>): ColumnNode {
   return ColumnNode.create(column as string)
 }
 
+export function parseOrderedColumnName(column: string): OperationNode {
+  const ORDER_SEPARATOR = ' '
+
+  if (column.includes(ORDER_SEPARATOR)) {
+    const [columnName, order] = column.split(ORDER_SEPARATOR).map(trim)
+
+    return parseOrderBy(columnName, order as any)
+  } else {
+    return parseColumnName(column as any)
```

Comment:
Doesn't `parseColumnName` accept a string? No cast needed here?

Reply:
True, I don't know why I used `as` there. Maybe the language server was f'ing up.